src/modules/gnd_pos_control/CMakeLists.txt

```diff
@@ -34,13 +34,10 @@ px4_add_module(
 	MODULE modules__gnd_pos_control
 	MAIN gnd_pos_control
 	STACK_MAIN 1200
-	COMPILE_FLAGS
 	SRCS
 		GroundRoverPositionControl.cpp
 	DEPENDS
-		platforms__common
+		external_lgpl
```

Comment:
We may want to aggregate the licenses of the files used in the final build so the user can comply with all the license requirements. In this case BSD + LGPL

Reply:
Yes if it comes to that, but we should push to get rid of these 600 lines of LGPL code.

Reply:
Paul has done a clean version that is almost ready to go.